doc/publish

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script builds the documentation and then publishes it to the web. See
+# the internal documentation for usage and how to set it up.
```

Comment:
Internal hear means it won't be on github?

Reply:
Yes.

Reply:
Yes.